crates/chia-bls/fuzz/fuzz_targets/blspy-fidelity.rs

```diff
@@ -15,14 +15,6 @@ use chia_bls::secret_key::SecretKey;
 use chia_bls::signature::{aggregate, sign};
 use pyo3::types::{PyBytes, PyList, PyTuple};
 
-fn to_bytes(obj: &PyAny) -> &[u8] {
-    obj.call_method0("__bytes__")
-        .unwrap()
-        .downcast::<PyBytes>()
-        .unwrap()
-        .as_bytes()
-}
-
```

Comment:
it's not obvious to me why you removed this function and inlined its code instead

Reply:
Fixed with PyBackedBytes